.github/workflows/server.yml

```diff
@@ -6,11 +6,30 @@ on:
     paths-ignore:
       - '**.md' #Do not need to run CI for markdown changes.
   pull_request:
-    branches: [ main, server-side ]
+    branches: [ main, server-side, cw/sc-206686/server-build-ci ]
```

Comment:
TODO: remove. Needed to run CI since I rebased this PR from `server-side` to the `server-build-ci` branch.